src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -88,6 +88,7 @@ public override void LogFunctionExecutionAggregateEvent(string siteName, string
 
         public override void LogFunctionExecutionEvent(string executionId, string siteName, int concurrency, string functionName, string invocationId, string executionStage, long executionTimeSpan, bool success)
         {
+          //  _metricsPublisher.AddFunctionExecutionActivity(functionName, invocationId, concurrency, executionStage, success, executionTimeSpan, DateTime.UtcNow);
```

Comment:
I don't think we need this

Reply:
the whole function, or just the body?

Reply:
never mind, i think the function is needed because of the interface.